data-raw/pull_data_scripts/TEMP_pull_from_edi.R

```diff
@@ -207,11 +197,48 @@ battle_clear_release_edi <- release_edi |>
                           site == "upper clear creek" ~ "ucc"),
          subsite = NA) |> 
   rename(origin = origin_released) |> 
-  left_join(dates) |> 
-  filter(date_released > max & year(date_released) > 2021) |> 
   select(date_released, release_id, stream, site, subsite, site_group,
          number_released, origin)
 
+# Deer/Mill ---------------------------------------------------------------
+catch_edi <- pull_edi("1504", 1, 3)
+recapture_edi <- pull_edi("1504", 2, 3)
+release_edi <- pull_edi("1504", 3, 3)
+trap_edi <- pull_edi("1504", 4, 3)
+
+deer_mill_catch_edi <- catch_edi |> 
+  rename(life_stage = lifestage) |> 
+  mutate(site = stream,
+         subsite = site,
+         site_group = stream)
+
+deer_mill_trap_edi <- trap_edi |> 
+  rename(trap_stop_date = date,
+         rpm_start = rpm_before,
+         rpm_end = rpm_after,
+         water_temp = water_temperature) |> 
+  mutate(site = stream,
+         subsite = site,
+         site_group = stream) |> 
+  select(trap_stop_date, stream, site, subsite, site_group, turbidity, water_temp, rpm_start, rpm_end)
+
+deer_mill_release_edi <- release_edi |> 
+  rename(date_release = release_date,
+         origin = release_origin) |> 
+  mutate(site = stream,
+         subsite = site,
+         site_group = stream) |> 
+  select(date_release, release_id, stream, site, subsite, site_group, number_released, origin)
+
+deer_mill_recapture_edi <- recapture_edi |> 
+  rename(date = recapture_date,
+         count = total_recaptured) |> 
+  mutate(site = stream,
+         subsite = site,
+         site_group = stream,
+         fork_length = mean_fl_recaptured) |> 
+  select(date, release_id, stream, site, subsite, site_group, count, fork_length)
+
 # Feather -----------------------------------------------------------------
 # Note there was an issue pushing these files to EDI due to large file size
```

Comment:
I can't access the Feather `.xlsx` files here

Reply:
I pushed these for the interim. Once we are able to connect to the db we wont need this.